lib/DBSQLOperation/index.ts

```diff
@@ -1,55 +1,40 @@
-import IOperation from './contracts/IOperation';
-import HiveDriver from './hive/HiveDriver';
+import IOperation, { IFetchOptions, defaultFetchOptions } from '../contracts/IOperation';
```

Comment:
This could be a total rookie question. But is there a reason we don't put all three names within the braces? 

```suggestion
import { IOperation, IFetchOptions, defaultFetchOptions } from '../contracts/IOperation';
```

Reply:
Because `IOperation` is a default export of the file, and I want it to match filename (`IOperation.ts`). Also, there are other places where only `IOperation` is imported, and doesn't look so weird there